### ISSUE TITLE
invalid value for Date Prompt will cause infinite loop

### ISSUE THREAD

**Issue by xhr0804** [microsoft/BotFramework-Composer#2583]
## Describe the bug
When I configure the default value as "30" for Date Prompt, it is a invalid value. But I this will cause an infinite loop when I chat with the bot.
![image](https://user-images.githubusercontent.com/44257569/78885945-04a4a600-7a90-11ea-8ce1-72bdf0e99217.png)
![image](https://user-images.githubusercontent.com/44257569/78886013-21d97480-7a90-11ea-8dee-c3b7152119b1.png)

## Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

**Comment by luhan2017** [comment]
@xhr0804, it is not related the value property, if you just type in 30 when it prompts, it will also let you to try again, because 30 is not recognized as a valued DateTime type, maybe you can try 30 mins or some other value string for DateTime.

**Comment by xhr0804** [comment]
@luhan2017 Yes, setting a valid value will be OK. It is just a minor problem. I just think if user types an invalid value, the dialog should either be able to collect the correct information during the conversation, or give an error message
